hw_3/task1.py

```diff
@@ -0,0 +1,12 @@
+n = int(input('Введите количество элементов: '))
+elements = input('Введите элементы через пробел: ').split()
+a_num = list(map(int, elements))
+if len(a_num) != n:
+    print('Введенные элементы не соответствуют заявленному количеству!')
+else:
+    x = int(input('Введите число X, которое необходимо найти в списке: '))
+    count = 0
+    for i in range(n):
```

Comment:
Может находиться напрямую по элементам массива.
```suggestion
    for i in a_num:
```